web/src/sidebar_ui.ts

```diff
@@ -465,6 +465,17 @@ export function initialize_right_sidebar(): void {
     );
 }
 
+function header_rows_selector(): string {
```

Comment:
Seems like this function should use a plural form, and maybe be prefixed with `get_`: `get_header_rows_selectors()`?

Reply:
Changed